recipes/isbnlib/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "isbnlib" %}
+{% set version = "3.10.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/isbnlib-{{ version }}.tar.gz
+  sha256: c9e6c1dcaa9dff195429373cf2beb3117f30b3fca43d7db5aec5a2d1f6f59784
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3
```

```suggestion
    - python >=3.6
```

But please double-check that.

Ref: https://github.com/xlcnd/isbnlib/blob/cd24d6af137d8c9ee95d75a3fdfe8377552f4a10/setup.cfg#L11